topics/transcriptomics/tutorials/rna-seq-counts-to-genes/tutorial.md

```diff
@@ -106,20 +106,20 @@ We will use three files for this analysis:
 >
 >     To import the files, there are two options:
 >     - Option 1: From a shared data library if available (ask your instructor)
```

Comment:
The snippet seems to be how to import as datasets not as a collection? or can I change that to collection somehow with `{{ include.astype | default: "as Datasets" }}` I had added a Tip box below that explains how to import from as collection but if you think it can be a snippet I could add that.